### DISCUSSION TITLE
App window location and size. Capture and restore.

### DISCUSSION THREAD

**Discussion by axtens** [unoplatform/uno#15933] [Q&A]
Windows 11; Uno4; C#; Visual Studio 2022 Community

how do I capture the position of the application windows on the monitor and the height/width of the application window and then apply same on relaunch?

**Comment by ArchieCoder** [comment]
@MartinZikmund Hi, ideally I would like this feature as well. You worked on Windowing stuff. Tagging you in case you know something exist.

**Comment by mobynet1** [comment]
@MartinZikmund I was looking for this as well. It was easy to do in WPF, but now I need to do this for any desktop implementation. I have asked my AI coding buddy, Codium AI, for some direction, but it was strictly a windows solution and not a good one at that. If there is a way, I really would like to find out how to do this.

Cheers,
Steve

**Comment by ArchieCoder** [comment]
@mobynet1 Check the thread in https://github.com/unoplatform/uno/issues/18348. Uno team recently introduced the support of AppWindow.
